### ISSUE TITLE
*.mesg_none instead of all 'No ... found' labels

### ISSUE THREAD

**Issue by enp** [facette/facette#284]
I have two facette 0.3.0 installations: on ubuntu 15.10 via deb and on RH-based system via files from deb extracted to /opt, so directory layout and configuration looks like:

```
$ find /opt/facette/ -type d
/opt/facette/
/opt/facette/conf
/opt/facette/conf/providers
/opt/facette/bin
/opt/facette/web
/opt/facette/web/template
/opt/facette/web/template/admin
/opt/facette/web/template/common
/opt/facette/web/template/show
/opt/facette/web/template/browse
/opt/facette/web/static
/opt/facette/web/static/fonts
/opt/facette/data

$ cat /opt/facette/conf/facette.json 
{
        "bind": ":1025",
        "base_dir": "/opt/facette/web",
        "providers_dir": "/opt/facette/conf/providers",
        "data_dir": "/opt/facette/data",
        "pid_file": "/opt/facette/facette.pid"
}
```

All works good on RH, but I see *.mesg_none instead of all 'No ... found' labels. All other labels looks good. Why can it be and how to fix it?

**Comment by vbatoufflet** [comment]
Hi @enp,

If you take a look at the web console in your browser. Do you see some error with the download of the `messages.json` file?

**Comment by enp** [comment]
No, this file is downloaded correctly and I see many "mesg_none" values in it, including "No unit found" for http://localhost:1025/admin/units/ and so.

**Comment by TomHetmer** [comment]
I have this bug too, but I didn't manage to find the cause.

If you could use a live install to debug you can try mine, for example
https://facette.hetmer.com/browse/collections/8cf1b87b-6ceb-4388-5fa6-bbc8213485bc
Clicking the refresh interval button shows an untranslated message.

Latest git master compiled for armhf, running behind caddy in http proxy mode.
Messages.json downloads fine there.

**Comment by vbatoufflet** [comment]
The last time I did encounter the issue was when the `Content-Type` returned along with the `messages.json` file wasn't `application/json`.

It seems to be the case here on your live instance @TomHetmer. The server returns a `application/octet-stream` type.

**Comment by TomHetmer** [comment]
Okay, that's the case.

https://github.com/facette/facette/blob/944d663717167dca92800e7f46e69bef08c13dcd/pkg/server/handler.go#L36

>func TypeByExtension
>The built-in table is small but on unix it is augmented by the local system's mime.types file(s) if available under one or more of these names:

>➜  ~  ls /etc/mime.types
>ls: /etc/mime.types: No such file or directory

That's what I get for running too minimalist systems (alpine linux there) :smile: 

Installed mailcap and now it's served fine.

**Comment by vbatoufflet** [comment]
Ok thanks for the pointing this out.

I wonder if it could be useful to use something like https://golang.org/pkg/mime/#AddExtensionType to enforce the JSON extension detection if not defined :smile:

**Comment by enp** [comment]
Thanks, after installing mailcap and adding application/json to mime.types all works fine.

**Comment by vbatoufflet** [comment]
Hi, this has been fixed in d77e77680d52d232305d0f12af6e3e1c857903cb.

The MIME type is now registered by default.
